runtime/src/bank.rs

```diff
@@ -1799,6 +1850,7 @@ impl Bank {
         start_slot_index: SlotIndex,
         end_slot_index: SlotIndex,
         epoch: Epoch,
+        auto_generated: bool,
```

Comment:
what does it mean exactly? Do we always pass true?

Reply:
Well, the naming was bad... I think I've made the intention clearer by some code changes and commenting..

This can be `true`/`false` in fact. This indicates given slot indexes are somewhat special ones or not. Namely, from this code block: https://github.com/solana-labs/solana/pull/10206/files#diff-a7549f152920d85fb44e6a784b8e5e1fR1843-R1869. And this corresponds to [this](https://github.com/solana-labs/solana/pull/10206/files#diff-a7549f152920d85fb44e6a784b8e5e1fR1919-R1920) in the ascii table.